website/pages/docs/features/color-mode.mdx

```diff
@@ -49,6 +49,26 @@ const theme = extendTheme({ config })
 export default theme
 ```
 
+For typescript, you need to explicitly describe the theme config type as `ThemeConfig` object.
+
+```jsx live=false
+// theme.tsx
```

Comment:
Use `theme.ts`

Reply:
Ok , I'll update this PR as soon as I figure out how

Reply:
Ok